.github/workflows/wheel.yaml

```
@@ -18,14 +30,25 @@ jobs:
     env:
       CIBW_TEST_COMMAND: python -c "import numcodecs"
       CIBW_BUILD: "cp310-* cp311-* cp312-* cp313-*"
-      CIBW_SKIP: "pp* *-musllinux_* *win32 *_i686 *_s390x"
+      CIBW_ARCHS_MACOS: native
+      CIBW_ARCHS_WINDOWS: native
+      CIBW_ARCHS_LINUX: "x86_64 aarch64"
       # note: CIBW_ENVIRONMENT is now set in pyproject.toml
 
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
 
+      - name: Restrict number of wheel builds on PRs
+        if: ${{ !env.BUILD_ALL }}
+        run: |
+          echo "CIBW_BUILD=cp310-win_amd64 cp311-manylinux_x86_64 cp312-macosx_x86_64 cp312-macosx_arm64" >> $GITHUB_ENV
```

Should we add Python 3.13 in this list? Perhaps using the other macOS build?
 
```suggestion
          echo "CIBW_BUILD=cp310-win_amd64 cp311-manylinux_x86_64 cp312-macosx_x86_64 cp313-macosx_arm64" >> $GITHUB_ENV
```